feat(blog) Upgrades blog & articles (#1468)
solana-foundation/solana-com: apps/media/content/upgrades/alpenglow.mdx

|                                                                                                                                                  |                                            |
| ------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------ |
| [SIMD-0326](https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0326-alpenglow.md)                             | Alpenglow Consensus Protocol               |
| [SIMD-0337](https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0337-parent-ready-update-marker.md)            | Markers for Alpenglow Fast Leader Handover |
| [SIMD-0357](https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0357-alpenglow_validator_admission_ticket.md)  | Alpenglow Validator Admission Ticket       |
| [SIMD-0384](https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0384-alpenglow-migration.md)                   | Alpenglow Migration                        |

> | [SIMD-0387](https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0387-bls-pubkey-management-in-vote-account.md) | BLS Pubkey Management in Vote Account      |

Reviewer: It would be nice to add some clarification on when these feature gates will ship.  There has been a lot of confusion about Alpenglow Validator admission ticket.  Both the BLS pubkey and alpenglow VAT are shipping in 4.1 .  Operators need to know that they must create their BLS key on chain before VAT is activated.

Author: added a clarification right under the Alpenglow SIMDs table: both the VAT (SIMD-0357) and BLS Pubkey Management (SIMD-0387) ship in Agave 4.1 (est. Q3 2026), and operators must register their BLS key ochain before the VAT feature gate activates or their validator won't be admitted

---

> title: Alpenglow Phase 1 - Votor

Reviewer 2: ```suggestion
title: Alpenglow
```

description: >-
  Alpenglow begins Solana's next consensus upgrade with Votor, targeting roughly
  150ms finality while replacing vote transactions with direct validator votes
  and aggregate certificates.
subtitle: Faster finality with Solana's next consensus protocol
publishedAt: 2026-06-24T00:00:00.000Z
status: published
author: solana-foundation
badges:
  - text: Under Development
    color: yellow
    variant: badge
metrics:
  - value: 150ms
    label: Target finality
  - value: 20+20
    label: Adversarial plus offline stake model
categories:
  - category: upgrades
tags:
  - tag: announcements
---

# Alpenglow Phase 1 - Votor

> **TBD** • Solana Foundation

Reviewer 2: ```suggestion
Q3 2026 • Solana Foundation
```

Solana is replacing its consensus protocol with Alpenglow, beginning with
Votor. Alpenglow aims to simplify Solana consensus at network scale, improve
performance, and provide faster finality. The target is roughly 150ms finality,
compared with Solana's current roughly 400ms pre-confirmation latency and
12.8-second TowerBFT finality.

A later Alpenglow phase is expected to replace Turbine with Rotor, a new block
propagation protocol.

|                                  |     |
| -------------------------------- | --- |

> | Expected Mainnet Activation Date | TBD |

Reviewer 2: ```suggestion
| Expected Mainnet Activation Date | Q3 2026 |
```

> | Devnet Activation                | TBD |

Reviewer 2: ```suggestion
| Devnet Activation                | Agave 4.3 |
```

| [SIMD-0326](https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0326-alpenglow.md)                             | Alpenglow Consensus Protocol               |
| [SIMD-0337](https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0337-parent-ready-update-marker.md)            | Markers for Alpenglow Fast Leader Handover |
| [SIMD-0357](https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0357-alpenglow_validator_admission_ticket.md)  | Alpenglow Validator Admission Ticket       |
| [SIMD-0384](https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0384-alpenglow-migration.md)                   | Alpenglow Migration                        |
| [SIMD-0387](https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0387-bls-pubkey-management-in-vote-account.md) | BLS Pubkey Management in Vote Account      |

**Shipping timeline:** Both the Alpenglow Validator Admission Ticket (VAT,
SIMD-0357) and BLS Pubkey Management in Vote Account (SIMD-0387) ship in Agave

> 4.1, estimated Q3 2026. Validator operators must create and register their BLS

Reviewer 2: ```suggestion
4.3, estimated Q3 2026. Validator operators must create and register their BLS
```

> key on-chain **before** the VAT feature gate is activated — otherwise their

Reviewer 2: ```suggestion
key onchain **before** the VAT feature gate is activated — otherwise their
```

validator will not be admitted under Alpenglow consensus.

---

### Phase 2 - Rotor

Alpenglow in its next phase is expected to replace Solana Turbine with a new
block propagation service called Rotor. It uses the same erasure coding idea and
validator stake-adjusted bandwidth from Turbine. Rotor changes the block
propagation plan from a tree of nodes to a single relay layer to minimize
network latency. The plan is to make this upgrade at a later stage once Votor
has already been adopted by the network.

## About This Upgrade

> Alpenglow is the biggest change to Solana in recent years. Not only does it

Reviewer 2: ```suggestion
Alpenglow represents a fundamental change to Solana's consensus layer, the first major consensus upgrade since TowerBFT. Not only does it
```
